src/part04.rs

```diff
@@ -49,7 +49,7 @@ fn ownership_demo() {
 //@ vector. Hence, when `vec_min` finishes, the entire vector is deleted. That's of course not what
 //@ we wanted! Can't we somehow give `vec_min` access to the vector, while retaining ownership of it?
 //@ 
-//@ Rust calls this *a reference* the vector, and it considers references as *borrowing* ownership. This
```

Comment:
I think what I wanted to write here is "to the vector", so that would be the appropriate fix.